projects/angular/src/data/_variables.tables.scss

```
@@ -5,46 +5,46 @@
-$clr-table-lineheightPaddingShim: (variables.$clr_baselineRem_1 - $clr-table-lineheight) * 0.5 !default;
-$clr-table-cellpadding: variables.$clr_baselineRem_0_5 !default;
-$clr-table-topcellpadding: $clr-table-cellpadding - variables.$clr_baselineRem_1px !default;
-$clr-table-bottomcellpadding: $clr-table-cellpadding - variables.$clr_baselineRem_1px !default;
-$clr-table-topcellpadding-cmp: $clr-table-cellpadding * 0.5 - variables.$clr_baselineRem_1px !default;
-$clr-table-bottomcellpadding-cmp: $clr-table-cellpadding * 0.5 - variables.$clr_baselineRem_1px !default;
-
-$clr-table--compact-rowHeight: variables.$clr_baselineRem_1 !default;
-$clr-table--compact-verticalPadding: ($clr-table--compact-rowHeight - $clr-table-lineheight) * 0.5 !default;
-
-// Table header styles
-$clr-thead-color: variables.$clr-p6-color !default;
-$clr-thead-font-size: variables.$clr-p6-font-size !default;
-$clr-thead-font-weight: variables.$clr-p6-font-weight !default;
-$clr-thead-letter-spacing: variables.$clr-p6-letter-spacing !default;
+$clr-table-borderwidth: var(--clr-table-borderwidth) !default;
+$clr-table-border-radius: var(--clr-table-border-radius) !default;
+$clr-table-cornercellradius: var(--clr-table-cornercellradius) !default;
+$clr-table-cell-padding: var(--clr-table-cell-padding) !default;
+$clr-table-cell-compact-padding: var(--clr-table-cell-compact-padding) !default;
+$clr-table-noborder-borderwidth: var(--clr-table-noborder-borderwidth) !default;
+$clr-table-vertical-align: var(--clr-table-vertical-align) !default;
+$clr-table-data-cell-font-size: var(--clr-table-data-cell-font-size) !default;
+$clr-table-data-cell-line-height: var(--clr-table-data-cell-line-height) !default;
+$clr-table-data-cell-font-weight: var(--clr-table-data-cell-font-weight) !default;
+$clr-table-header-cell-font-size: var(--clr-table-header-cell-font-size) !default;
+$clr-table-header-cell-line-height: var(--clr-table-header-cell-line-height) !default;
+$clr-table-header-cell-font-weight: var(--clr-table-header-cell-font-weight) !default;
+
+$clr-table-bgcolor: var(--clr-table-bgcolor) !default;
+$clr-thead-bgcolor: var(--clr-thead-bgcolor) !default;
+
+$clr-table-header-border-bottom-color: var(--clr-table-header-border-bottom-color) !default;
+$clr-table-footer-border-top-color: var(--clr-table-footer-border-top-color) !default;
+$clr-table-border-color: var(--clr-table-border-color) !default;
+
+$clr-tablerow-bordercolor: var(--clr-tablerow-bordercolor) !default;
+
+$clr-table-font-color: var(--clr-table-font-color) !default;
+$clr-table-margin: var(--clr-table-margin) !default;
+
+$clr-thead-color: var(--clr-thead-color) !default;
+
+$clr-table-smallpadding: var(--clr-table-smallpadding) !default;
+$clr-table-bottomcellpadding: var(--clr-table-bottomcellpadding) !default;
+$clr-table-topcellpadding: var(--clr-table-topcellpadding) !default;
+$clr-table-fontsize: var(--clr-table-fontsize) !default;
+$clr-table-lineheight: var(--clr-table-lineheight) !default;
```

You can mark as deprecated the property as well.

```suggestion
// @deprecated not used for line height calculation, but for vertical padding calculation only
$clr-table-lineheight: var(--clr-table-lineheight) !default;
```